Sources/Crypto/RSA/RSAPadding.swift

```diff
@@ -0,0 +1,49 @@
+import CNIOOpenSSL
+import Foundation
+
+/// RSA Paddings
+public enum RSAPadding: Int32 {
+    /// PKCS #1 v1.5 padding
+    case pkcs1
+    /// SSLv23 padding
+    case sslv23
+    /// No padding
+    case none
+    /// PKCS #1 v2.0 EME-OAEP + SHA-1 + MGF1 padding
+    case pkcs1_OAEP
+    /// X9.31 padding
+    case x931
+
+    public init?(rawValue: Int32) {
+        switch rawValue {
+        case RSA_PKCS1_PADDING:
+            self = .pkcs1
+        case RSA_SSLV23_PADDING:
+            self = .sslv23
+        case RSA_NO_PADDING:
+            self = .none
+        case RSA_PKCS1_OAEP_PADDING:
+            self = .pkcs1_OAEP
+        case RSA_X931_PADDING:
+            self = .x931
+        default:
+            return nil
+        }
+    }
+
+    public var rawValue: Int32 {
```

Comment:
I don't think you need to implement this method here. You should be able to use the `=` syntax when declaring the enum

Reply:
This method is necessary otherwise the underlying rawValues would be from 0-4, but the OpenSSL values are different. Sadly one can't assign the OpenSSL values directly, because they aren't static. Without this method this would be false:

RSAPadding(rawValue: RSA_PKCS1_PADDING).rawValue == RSA_PKCS1_PADDING

Reply:
You should be able to achieve by doing something like this:
```swift
public enum RSAPadding: Int32 {
    case pkcs1 = RSA_PKCS1_PADDING
    ...
}
```

I will take a look at adding this on master once merged, thanks!

Reply:
(Nevermind, you were right! Didn't understand the part about literals)